docs/features.md

```diff
@@ -28,7 +28,7 @@ The SpatialOS Unreal GDK also supports:
 
 ## Static component replication
 
-The SpatialOS Unreal GDK supports only one instance of each type of static component replication for components owned by an Actor. For example, an Actor can have only one FooComponent and one BarComponent; it can’t have two FooComponents or two BarFooComponents. 
+The SpatialOS Unreal GDK supports only one instance of each type of static component replication for components owned by an Actor. For example, an Actor can have only one `FooComponent` and one `BarComponent`; it can’t have two `FooComponent`s or two `BarFooComponent`s. 
```

Comment:
I added back-ticks; hopefully they're correct

Reply:
Oh there is a typo. Can you change the `BarFooComponent` to `BarComponent`.